variables.tf

```diff
@@ -35,6 +35,12 @@ variable "lock_resources" {
 
 # Networking
 
+variable "graphdb_external_address_fqdn" {
```

Comment:
BTW, FQDN is the wrong term here, FQDNs do not include protocol, port, path etc.
Consider `graphdb_external_address`

Reply:
Precisely, for example: the variable value is just `graphdb.duckdns.org` no protocol or port.
in `/etc/graphdb-cluster-proxy/graphdb.properties` we have `graphdb.external-url=https://${graphdb_external_address_fqdn}`

Reply:
Huh, it's ok then.